code/processes/wdb.q

```diff
@@ -147,11 +171,46 @@ savetablesbypart:{[dir;pt;forcesave;tablename]
 		if[gc;.gc.run[]];
 	];
 	};
+
+savetablesbypartenum:{[dir;pt;forcesave;tablename]
+	savetablesbypartenumcol[dir;pt;forcesave;tablename;enumcol];
+ };
```

Comment:
would it make sense to just have this be a projection?

```suggestion
savetablesbypartenum:savetablesbypartenumcol[;;;;enumcol];
```

(defined after `savetablesbypartenumcol`)